idmapping-rest/src/main/java/org/uniprot/api/idmapping/controller/validator/IdMappingDownloadRequestValidatorFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.uniprot.api.idmapping.controller.validator;
+
+import org.uniprot.api.common.exception.InvalidRequestException;
+
+public class IdMappingDownloadRequestValidatorFactory {
+
```

Comment:
I think i did not completely follow your comment.
We already validate fields and format in the validation (UniProtKBIdMappingDownloadRequestValidator, UniParcIdMappingDownloadRequestValidator and UniRefIdMappingDownloadRequestValidator ), and we are validating against kb, ref and parc field config accordingly.

Reply:
instead of introducing new string for each to type in case, use to value from IdMappingFieldConfig 
``` switch (type.toLowerCase()) {
            case "uniprotkb":
                validator = new UniProtKBIdMappingDownloadRequestValidator();
                break;
            case "uniparc":
                validator = new UniParcIdMappingDownloadRequestValidator();
                break;
            case "uniref50":
            case "uniref90":
            case "uniref100":```